sdks/typescript/package.json

```diff
@@ -3,17 +3,34 @@
   "version": "0.0.1",
   "type": "module",
   "license": "Apache-2.0",
-  "author": "diconium",
+  "author": "Diconium Digital Solutions",
   "description": "Ethereal Pulse Typescript SDK",
   "homepage": "https://github.com/diconium/ethereal-pulse#readme",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/diconium/ethereal-pulse.git"
+    "url": "git+https://github.com/diconium/ethereal-pulse.git",
+    "directory": "sdks/typescript"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "build": "tsc",
+    "prepublishOnly": "npm run build"
   },
+  "keywords": [
+    "ethereal",
+    "pulse",
+    "sdk",
+    "email"
+  ],
   "devDependencies": {
-    "typescript": "^5.4.5"
-  }
+    "jest": "^29.7.0",
+    "ts-jest": "^29.1.4",
+    "typescript": "^5.4.5",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^20.14.2"
+  },
+  "dependencies": {
+    "axios": "^1.7.2"
+  },
```

Comment:
For our use case using axios is a bit overkill and we could just use fetch instead

Reply:
This typescript version I think could also be used in Node.js projects instead, only client-based projects. In this case http `fetch` method, isn't available in nodejs, due this I suggest use `axios`.

Reply:
Fetch is available in Node and stable since v21 so I don't think that would be an issue

Reply:
Good to know thank you. I will update this one.